api/src/main/resources/messages.properties

```diff
@@ -1,4 +1,8 @@
 ${project.parent.artifactId}.title=Bahmni Appointment Scheduling
 
 bahmni.appointment.teleConsultation.patientEmailNotificationSubject=Teleconsultation appointment confirmed
-bahmni.appointment.teleConsultation.patientEmailNotificationTemplate=Hi {0}, Your tele-consultation appointment with {1} for {2} has been scheduled! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!
\ No newline at end of file
+bahmni.appointment.teleConsultation.patientEmailNotificationTemplate=Hi {0}, Your tele-consultation appointment with {1} for {2} has been scheduled! Please use this link {3}. If you have any questions, please reach out to administration for assistance. See you soon!
+
+bahmni.appointment.adhocTeleConsultation.patientEmailNotificationSubject=AdhocTeleconsultation started

```

Comment:
We should remove the word "Adhoc" from the email, because we don't know if its really Adhoc or not. It may be planned by phone, and then clicked by Doctor (without using the appointment module). It should just say, your Teleconsultation session has started. To join, please use the link: [link].